scenes/game/local_game.gd

```diff
@@ -3234,6 +3245,18 @@ func is_effect_condition_met(performing_player : Player, effect, local_condition
 			return true
 		elif condition == "opponent_at_edge_of_arena":
 			return other_player.is_at_edge_of_arena()
+		elif condition == "opponent_at_max_range":
+			assert(active_strike)
+			var attack_card = active_strike.get_player_card(performing_player)
+			var max_range = get_card_stat(performing_player, attack_card, 'range_max') + performing_player.strike_stat_boosts.max_range
```

Comment:
probably a good candidate for a get_total_max_range function. And then technically we should probably overwrite range in attack removal cleanup.
But does this matter for any characters? Not sure if anyone cares about the range of their opponent's attack?
Maybe don't bother if nobody cares.

Reply:
Maybe carl? but only if the attack could somehow be removed before it hit? but yeah might at least make get_total_max_range, and probably get_attack_origin while i'm at it